src/index.js

```diff
@@ -142,6 +143,9 @@ if (window.CanvasRenderingContext2D) {
       const tool = docViewer.getTool(window.Tools.ToolNames.STICKY);
       tool?.setSaveViewState(true);
     }
+    docViewer.on('documentLoaded', () => {
```

Comment:
Do we need to do this on every document load or can we do it just once initially?

Reply:
My understanding (from what @ZhijieZhang taught me): before we load a new document, we will call `docViewer.closeDocument` to close the current document, and `docViewer.closeDocument` will trigger the following functions call chain:
In file `DocumentViewer.ts` when called `closeDocument()` this will trigger -> `dispose()` in the same file, then trigger -> `annotationManager.dispose()` (in AnnotationManager.js), then trigger -> `this.init()` (in file `FieldManager.js`), and this will set my printHandler to null; 

This is why my puppeteer test failed (since printHandler = null).

Reply:
Move setPrintHandler() to here